### DISCUSSION TITLE
galaxy watch 4 standalone

### DISCUSSION THREAD

**Discussion by dlinah** [Navid200/xDrip#177] [Watches]
i've been told that the  g4's xdrip integration does not work as a standalone, is there any workaround to get it to work?
if not, is it possible for it to be supported in the future or is it a hardware issue?
unfortunately this is the only wear os watch i can find in my country

**Comment by Navid200** [comment]
Do you have the watch already?

**Reply by dlinah** [reply]
no not yet

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
